Code/GraphMol/MolOps.h

```diff
@@ -243,13 +243,14 @@ void mergeQueryHs(RWMol &mol, bool mergeUnmappedOnly = false);
 
 typedef enum {
   ADJUST_IGNORENONE = 0x0,
-  ADJUST_IGNORECHAINATOMS = 0x1,
-  ADJUST_IGNORERINGATOMS = 0x4,
+  ADJUST_IGNORECHAINS = 0x1,
```

Comment:
This piece of API does break, but it was only partially present in the previous release anyway. There are very few pieces of the rest of the code affected